app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -59,15 +70,19 @@
 
     public String errorDesc;
 
-    private DrawerLayout mDrawerLayout;
+//    private DrawerLayout mDrawerLayout;
 
     private Toolbar mToolbar;
 
-    private NavigationView navigationView;
+//    private NavigationView navigationView;
+//
+//    private ProgressBar downloadProgress;
+//
+//    private CoordinatorLayout mainFrame;
```

Comment:
I'm still in the process of migrating the code for the old design to the new one